eng/scripts/python/apiview-syncdb/sync_cosmosdb.py

```diff
@@ -25,13 +26,14 @@
 BACKUP_CONTAINER = "backups"
 BLOB_NAME_PATTERN ="cosmos/{0}/{1}"
 
+# Create a DefaultAzureCredential instance
+credentials = DefaultAzureCredential()
```

Comment:
If you are running in a DevOps agent they will have a MI on the machine without the correct permission so you will not reach the AzPS credential which comes after it in the chain. We have to disable MI or create an explicit chained credential in other places.